sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -60,6 +60,16 @@ static const bool UseCopyEngineForD2DCopy = [] {
   return (CopyEngineForD2DCopy && (std::stoi(CopyEngineForD2DCopy) != 0));
 }();
 
+// This is an experiemental option that allows the use of copy engine, if
+// available in the device, in Level Zero plugin for copy operations submitted
+// to an in-order queue. The default is 0.
```

Comment:
Is it intentionally disabled by default?

Reply:
-  bool UseCopyEngine = PreferCopyEngine && Queue->Device->hasCopyEngine();
+  bool UseCopyEngine = (!(Queue->isInOrderQueue()) ||
+                        UseCopyEngineForInOrderQueue) && PreferCopyEngine &&
+                        Queue->Device->hasCopyEngine();
 
@smaslov-intel  @asudarsa   Arvind, did you miss above code in the merge?

Reply:
Hi @xtian-github
Can you please comment if we want to keep this option turned 'on' or 'off' by default?
And, yes, it does look like I missed these lines during the patch creation. I will fix it now. Thanks

Reply:
@asudarsa  for this new experimental ENV, it should be default OFF, as I did in the patch I give you. We will use it for performance tuning and figure perf regression. For the original COPY engine ENV, it should be default ON (no change to it).

Reply:
We aren't adding NEW env var, we are extending existing one. Let's have all copy engines ON by default and let people use env var to scale it back to deal with any possible regressions

Reply:
Maybe we can make it ON by default and have people use env var to deal with regressions?

Reply:
I would be OK with it. My worry is people may be surprised with regressions.  What is the extension for the syntax?

Reply:
I have marked it ON by default. If the user does not specify it, copy engine will be used for in-order queue copies.